blosc/dframe.c

```diff
@@ -37,7 +37,7 @@
 
 
 /* Append an existing chunk into an extended frame. */

```

Comment:
extended - > disperse

Reply:
Fixed in 2426e5ae2a82ab94f3fc05902aeb22df78beb0b3 .